dist/ktx2Decoder/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "author": {
+        "name": "David CATUHE"
```

Comment:
I think you deserve to be named here ;)

Reply:
Well, I don't care and it's a collective effort: your name is somewhat a placeholder for the whole team ;)

Reply:
LOL makes sense!